gitapp/lib/main.dart

```diff
@@ -46,19 +42,82 @@ class MyHomePage extends StatefulWidget {
   _MyHomePageState createState() => _MyHomePageState();
 }
 
-class _MyHomePageState extends State<MyHomePage> {
-  int _counter = 0;
+class _MyHomePageState extends State<MyHomePage> with TickerProviderStateMixin {
+  //FGA-3
+  final List<UserCard> _card = <UserCard>[];
+  final UsersList users = new UsersList();
+  String name = "";
+  final TextEditingController _textController = new TextEditingController();
+  bool searching =  false;
+
+  Future _getUsers(String text) async {
 
-  void _incrementCounter() {
+    _card.clear();
     setState(() {
-      // This call to setState tells the Flutter framework that something has
-      // changed in this State, which causes it to rerun the build method below
-      // so that the display can reflect the updated values. If we changed
-      // _counter without calling setState(), then the build method would not be
-      // called again, and so nothing would appear to happen.
-      _counter++;
+      searching = true;
     });
+   if(text!="")
```

Comment:
Please fix the formatting.
Prefer `if () {` so the braces should be in the same line

Reply:
solved